.github/workflows/project.yml

```diff
@@ -13,32 +13,40 @@ concurrency:
 
 jobs:
   integration:
+    strategy:
+      matrix:
+        version: [14, 16, 18]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: ./.github/actions/node/setup
       - run: yarn install
-      - uses: ./.github/actions/node/14
-      - run: yarn test:integration
-      - uses: ./.github/actions/node/16
-      - run: yarn test:integration
-      - uses: ./.github/actions/node/18
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.version }}
```

Comment:
`latest` is supported as a value here, so the second job is not needed.

Reply:
ok I assumed we wanted `latest` to mean whatever we defined in `.github/actions/node/latest/action.yml`, but if we're ok with using `latest` here, that's perfect